tests/unit/test_wallet_ethereum.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import asdict, dataclass
+
+import pytest
+
+from aleph.sdk.chains.common import get_verification_buffer
+from aleph.sdk.chains.ethereum import verify_signature
+from aleph.sdk.exceptions import BadSignatureError
+from aleph.sdk.wallets.ledger.ethereum import LedgerETHAccount, get_fallback_account
+
+
+@dataclass
+class Message:
+    chain: str
+    sender: str
+    type: str
+    item_hash: str
+
+
+@pytest.mark.ledger_hardware
+@pytest.mark.asyncio
+async def test_LedgerETHAccount():
```

Comment:
Nitpick: snake case?

```suggestion
async def test_ledger_eth_account():
```